src/extensions/group-by-v2/bootstrap-table-group-by.js

```diff
@@ -190,9 +190,16 @@ BootstrapTable.prototype.initBody = function (...args) {
     if (this.options.groupByToggle) {
       this.$container.off('click', '.groupBy')
         .on('click', '.groupBy', function () {
+          const groupIndex = $(this).closest('tr').data('group-index')
+
           $(this).toggleClass('expanded collapsed')
           $(this).find('span').toggleClass(`${that.options.icons.collapseGroup} ${that.options.icons.expandGroup}`)
-          that.$body.find(`tr[data-parent-index=${$(this).closest('tr').data('group-index')}]`).toggleClass('hidden')
+
+          that.$body.find(`tr[data-parent-index=${groupIndex}]`).toggleClass('hidden')
```

Comment:
I think also that we can save this in a const variable and use it here and in the next line